index.js

```diff
@@ -186,34 +207,160 @@ app.post('/quote', (req, res) => {
             ]
           }
         ]
-      }
-      )
+      })
   }
 
 })
 
 
-app.post('/api/response', (req, res) => {
+app.post('/api/response', async (req, res) => {
   const parsedPayload = JSON.parse(req.body.payload)
   if (parsedPayload.actions[0].value === 'cancel_quote') {
     res.sendStatus(200)
     request.post({
-      headers: { 'content-type': 'application/json' },
+      headers: {
+        'content-type': 'application/json'
+      },
       uri: parsedPayload.response_url,
       body: JSON.stringify({
         "delete_original": "true"
       })
-    }
-    )
+    })
+  } else if (parsedPayload.actions[0].value.slice(0,15) === 'get_more_quotes') {
+    // GET QUOTES FROM DB
+    let quotes = await dbQuery('something')
+    console.log(req.body)
+    res.status(200)
+    request.post({
+      headers: {
+        'content-type': 'application/json',
+      },
+      uri: parsedPayload.response_url,
+      body: JSON.stringify({
+        "blocks": [{
+            "type": "section",
+            "text": {
+              "type": "mrkdwn",
+              "text": `Here are some quotes we found matching "${parsedPayload.actions[0].value.slice(16)}"`
+            }
+          },
+          {
+            "type": "divider"
+          },
+          {
+            "type": "section",
+            "text": {
+              "type": "mrkdwn",
+              "text": quotes[0].quote
+            },
+            "accessory": {
+              "type": "button",
+              "text": {
+                "type": "plain_text",
+                "emoji": true,
+                "text": "Pick Me"
+              },
+              "value": "pick_option_1"
+            }
+          },
+          {
+            "type": "context",
+            "elements": [{
+              "type": "mrkdwn",
+              "text": `Quote by ${quotes[0].character} from The Lord of the Rings`
```

Comment:
Same situation here as the comment from up above. It may be good to have the following objects created by a function dynamically.